UEBPrep/spec/controllers/user_controller_spec.rb

```diff
@@ -68,13 +68,13 @@
 
     describe '#update' do
       it 'updates the user with valid new data' do
-        patch :update, :id => @user.id, :email => 'new@new.com', :format => :json
+        patch :update, :id => @user.id, :user => { :email => 'new@new.com' }, :format => :json
```

Comment:
reflects changes in strong params, we must pass in an object that has the name :user